database/migrations/2024_04_24_173610_create_language_resume_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('language_resume', function (Blueprint $table) {
+            $table->id();
+            $table->foreignUuid('language_id')->references('id')->on('languages')->onDelete('cascade');
+            $table->foreignUuid('resume_id')->references('id')->on('resumes')->onDelete('cascade');
```

Comment:
Antonio I have passed you a comment on Discord about this line.

Reply:
Thanks for your comment!!

Reply:
🤔